Guared1365/packs copy

Guared1365 opened the pull request:
<!-- ЗДЕСЬ должно быть **подробное описание** того, что происходит в PR и зачем это нужно. PR не должен содержать изменений, о которых здесь ничего не сказано. -->
Изменил положение навигационного огня на обшивке Сьерры, так как он находился на инжекторе. Добавил 3 сканера аномалий, диски для автолата и доску для бумаг на авейку Восхождения (CaulShip).

<!-- ЗДЕСЬ нужно **привязать ишью**, которые относятся к PR'у в формате `close #1234` или `fixes #1234` - тогда они автоматически закроются вместе с PR. Можно написать `Затрагивает #1234, но не фиксит его`, если вы просто хотите упоминуть ишью, но не закрывать его. -->

<!-- ЗДЕСЬ нужно **расписать изменения** которые попадут в **чейнджлог**, формат - `prefix: краткое описание` -->
### Чейнджлог
```yml
🆑Guared1365
maptweak: Изменил положение навигационного огня на обшивке Сьерры, так как он находился на инжекторе. Добавил 3 сканера аномалий, диски для автолата и доску для бумаг на авейку Восхождения (CaulShip).
/🆑
```

<!--
  Честно заполняем галочки. Чем больше галочек, тем быстрее проверять Pull Request, соответственно он быстрее будет принят.
  Чтобы отметить - ставим `x` (икс) внутри квадратных скобочек вот так: `- [x] ...`.
  Галочки можно доставлять позже по мере окончания работы над PR'ом.
-->

- [x] Pull Request полностью завершен, мне не нужна помощь чтобы его закончить.
- [x] Я внимательно прочитал все свои изменения и багов в них не нашел.
- [x] Я запускал сервер со своими изменениями локально и все протестировал.

Guared1365 committed:
Малое изменение карты Сьерры

Один из навигационных огней снаружи Сьерры находился на инжекторе. Некрасиво и неудобно. Переместил его в другое место.

Guared1365 committed:
Изменение Caulship'а Восхождения

Добавил доску для прикрепления отчётов о сканировании (не нужно будет складывать их на пол). В связи с тем, что автолаты у нас теперь требуют дисков, добавил и их. Также добавил сканеры аномалий (3 штуки), так как те, что находятся в ИКСах не работают.

LordNest reviewed (dismissed):
Уже есть активный ПР с изменением карты Сиерры. Не мерджить до мерджа

AmShegars committed:
Merge branch 'dev-sierra' into guared1365/packs-copy

Guared1365 committed:
Merge branch 'dev-sierra' into guared1365/packs-copy

LordNest committed:
Merge branch 'dev-sierra' into guared1365/packs-copy

Guared1365 committed:
Изменение карты Сьерры №2

Изменил расположение навигационного огня, так как тот налезал на спрайт инжектора.

Guared1365 committed:
Изменение карты Сьерры

Изменил месторасположение навигационного огня
